code/datums/components/storage/concrete/bag_of_holding.dm

```diff
@@ -15,13 +15,29 @@
 	if(safety != "Proceed" || QDELETED(A) || QDELETED(W) || QDELETED(user) || !user.canUseTopic(A, BE_CLOSE, iscarbon(user)))
 		return
 	var/turf/loccheck = get_turf(A)
-	to_chat(user, span_danger("The Bluespace interfaces of the two devices catastrophically malfunction!"))
+	if(istype(loccheck.loc, /area/fabric_of_reality))
+		to_chat(user, "<span class='warning'>You can't do that here!</span>")
+		return
+	to_chat(user, "<span class='danger'>The Bluespace interfaces of the two devices catastrophically malfunction!</span>")
 	qdel(W)
-	playsound(loccheck,'sound/effects/supermatter.ogg', 200, TRUE)
+	playsound(loccheck,'sound/effects/supermatter.ogg', 200, 1)
 
 	message_admins("[ADMIN_LOOKUPFLW(user)] detonated a bag of holding at [ADMIN_VERBOSEJMP(loccheck)].")
 	log_game("[key_name(user)] detonated a bag of holding at [loc_name(loccheck)].")
 
 	user.gib(TRUE, TRUE, TRUE)
-	new/obj/boh_tear(loccheck)
+	for(var/turf/T in range(6,loccheck))
+		if(istype(T, /turf/open/space/transit))
+			continue
+		for(var/mob/living/M in T)
+			if(M.movement_type & FLYING)
+				M.visible_message("<span class='danger'>The bluespace collapse crushes the air towards it, pulling [M] towards the ground...</span>")
```

Comment:
wording is kindof weird, perhaps 'the bluespace collapse knocks you to the ground?' or something

im not hard over on that wording just feels weird to me

Reply:
I like how it sounds because it's saying the air gets rapidly sucked up and pushes you in